tests/apps/jsonlayerstest.cpp

```diff
@@ -341,6 +342,7 @@ static void fill_hwclayer(hwcomposer::HwcLayer *pHwcLayer,
       pParameter->frame_x, pParameter->frame_y, pParameter->frame_width,
       pParameter->frame_height));
   pHwcLayer->SetNativeHandle(pRenderer->GetNativeBoHandle());
+  pHwcLayer->release_fence.Reset(-1);
 }
 
 static void init_frames(int32_t width, int32_t height) {
```

Comment:
Let have one demo which is single buffered and we can continue looking at how to improve the situation there. For rest lets move to double buffered cases. Blocking on current fence even though is fixing tearing but shows lot of lag :(